.github/scripts/run-fips-ut.sh

```diff
@@ -16,4 +20,11 @@ export JAVA_HOME=/etc/alternatives/java_sdk_17
 if [ $? -ne 0 ]; then
   exit 1
 fi
-./mvnw test -nsu -B -pl crypto/default,crypto/fips1402 -Dcom.redhat.fips=true -Dorg.bouncycastle.fips.approved_only=true
+
+# NOTE the use of  "org.bouncycastle.rsa.allow_pkcs15_enc" as per BCFIPS release notes:
+#
+# End of 2023 transition for RSA PKCS1.5 encryption. The provider blocks RSA with PKCS1.5 encryption.
+# The following property can be used to override the default behavior:
+# org.bouncycastle.rsa.allow_pkcs15_enc (allow use of PKCS1.5)
+# This is required by crypto/fips1402/src/test/java/org/keycloak/crypto/fips/test/FIPS1402JWETest.java
+./mvnw test -nsu -B -pl crypto/default,crypto/fips1402 -Dcom.redhat.fips=true -Dorg.bouncycastle.fips.approved_only=true -Dorg.bouncycastle.rsa.allow_pkcs15_enc=true
```

Comment:
Same here, bc-fips 1.0.2.4 `deals with the end of the transition periods for PKCS 1.5 RSA encryption and TDES encryption`. See release notes again. I would not add the option unconditionally because it will disappear in the future, in my PR I have skipped those methods is strict enabled and changed another test to use another allowed encryption method (RSA-OAEP-256). Documentation also added.